keps/sig-cluster-lifecycle/kubeadm/0023-documentation-for-images.md

```diff
@@ -1,5 +1,3 @@
-
```

Comment:
this is not a KEP its just some sequence diagrams that fall under "proposal".
it's also out of date.

i'd say just delete this file.

if someone wants to they can move it to the kubernetes/kubeadm repository under docs.

Reply:
could you also move the file https://github.com/kubernetes/enhancements/blob/a80c69127e965a7c90fbbae70fc2c752c7ae008b/keps/sig-cluster-lifecycle/kubeadm/README.md to the parent sig-cluster-lifecycle folder.

Reply:
moved to the parent folder.